src/client/shadows/dynamicshadowsrender.cpp

```diff
@@ -292,7 +295,6 @@ void ShadowRenderer::updateSMTextures()
 				// This is also handled in ClientMap.
 				if (m_current_frame == m_map_shadow_update_frames - 1 || m_force_update_shadow_map) {
 					if (m_shadow_map_colored) {
-						m_driver->setRenderTarget(0, false, false);
```

Comment:
Was that unnecessary before? Colored shadows still work, so I suppose the answer it yes.